Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.py

```diff
@@ -1710,6 +1933,224 @@ def report_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    group_name = args.get("group_name", "")
+    dictionary_name = args.get("dictionary_name", "")
+
+    host_name, group_name = check_dictionary_mode_args(mode, host_name, group_name)
+
+    response = client.dictionary_delete_request(dictionary_name=dictionary_name,
+                                                mode=mode,
+                                                host_name=host_name,
+                                                group_name=group_name
+                                                )
+
+    return CommandResults(
+        readable_output=f"{dictionary_name} deleted successfully.",
+        raw_response=response,
+    )
+
+
+def dictionary_words_add_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Add words to an existing dictionary configuration.
+
+    Args:
+        client (Client): Cisco ESA API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: Readable outputs for XSOAR, indicating the successful addition of words to the dictionary.
+    """
+    mode = args.get("mode", DEFAULT_MODE_DICTIONARIES)
+    host_name = args.get("host_name", "")
+    group_name = args.get("group_name", "")
+    dictionary_name = args.get("dictionary_name", "")
+    words = args.get("words", "")
+
+    host_name, group_name = check_dictionary_mode_args(mode, host_name, group_name)
+    words = convert_words_to_list(words)
+
+    response = client.dictionary_words_add_request(dictionary_name=dictionary_name,
+                                                   mode=mode,
+                                                   host_name=host_name,
+                                                   group_name=group_name,
+                                                   words=words
+                                                   )
+
+    return CommandResults(
+        readable_output=f"Added successfully to {dictionary_name}.",
+        raw_response=response,
+    )
+
+
+def dictionary_words_delete_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Delete words from an existing dictionary configuration.
+
+    Args:
+        client (Client): Cisco ESA API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: Readable outputs for XSOAR, indicating the successful deletion of words from the dictionary.
+    """
+    mode = args.get("mode", DEFAULT_MODE_DICTIONARIES)
+    host_name = args.get("host_name", "")
+    group_name = args.get("group_name", "")
+    dictionary_name = args.get("dictionary_name", "")
+    words = args.get("words", "")
+
+    host_name, group_name = check_dictionary_mode_args(mode, host_name, group_name)
+    try:
+        words = ast.literal_eval(words)
+    except Exception:
+        DemistoException("Words list is not defined correctly. Please use the following pattern: ['word1','word2','word3'].")
```

Comment:
You don't need this

Reply:
Please see my previous comment on argToList